.kitchen.yml

```diff
@@ -36,6 +36,18 @@ suites:
           backend: local
     provisioner:
       name: terraform
+  - name: "disable_client_cert"
+    driver:
+      name: "terraform"
+      command_timeout: 1800
+      root_module_directory: test/fixtures/disable_client_cert
+    verifier:
```

Comment:
Could you add `color: false` here, per https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/58?